src/pages/transactions/TransactionForm.js

```diff
@@ -133,7 +138,9 @@ export default function TransactionForm({ onAddTransaction, onAlert }) {
           </select>
         </div>
         {/* Submit button */}
-        <button type="submit">Add</button>
+        <Button variant="primary" endIcon="add" onClick={onAddTransaction}>
+          Add
+        </Button>
       </form>
     </div>
   );
```

Comment:
QA issues: you have the following error in your console, when you open the project locally: 
Warning: Received `true` for a non-boolean attribute `end`.
If you want to write it to the DOM, pass a string instead: end="true" or end={value.toString()}.